ostree-simplified-installer.sh

```diff
@@ -447,6 +553,19 @@ for LOOP_COUNTER in $(seq 0 30); do
     sleep 10
 done
 
+# FDO test case: check if /boot/device-credentials exist.
```

Comment:
This test is duplicated. If the credential does not exist, the installation will be failed. Checking this one after installation does not make any sense.